### PR TITLE
Update EIP-1202: Resolve new EIPW and bring this EIP back to live

### PR DESCRIPTION
**Pull request by xinbenlv**

### CODE REVIEW

File: EIPS/eip-1202.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Voting Standard
 author: Zainan Victor Zhou (@xinbenlv), Evan (@evbots), Yin Xu (@yingogobot)
 type: Standards Track
 category: ERC
-status: Draft
+status: Last Call
```
Comment (MicahZoltu) [inline EIPS/eip-1202.md:7]:
```suggestion
status: Review
```
The step after Draft is review.

Comment (xinbenlv) [reply EIPS/eip-1202.md]: Thanks for the suggestion. Done

**Comment by xinbenlv** [comment]
@MicahZoltu I updated the status to `review`, can you help check if there is anything else I need to do to move this PR forward?

**Comment by MicahZoltu** [comment]
I don't handle ERCs, so you'll have to wait for another editor to get to reviewing this EIP.  I left the comment just to hopefully save a round-trip with the reviewer later since I happened to notice it.

**Review by lightclient — CHANGES_REQUESTED** [review]
Please remove the "Bibliography" and "Simple Code Example" sections. This are not valid EIP sections per EIP-1.

**Comment by xinbenlv** [comment]
@lightclient thank you, I removed these sections per your feedback.

File: EIPS/eip-1202.md [hunk 2]
```diff
@@ -113,29 +99,6 @@ The third consideration is non-trivialness. Some voting applications assume ***a
...
-
-### Worthnoting Projects
- - [Ethereum DAO: How to build a DEMOCRACY on the blockchain](https://www.ethereum.org/dao)
- - [Carbon Vote](http://carbonvote.com/)
- - [Paper: A Smart Contract for Boardroom Voting with Maximum Voter Privacy](https://eprint.iacr.org/2017/110.pdf) - *Suggested by @aodhgan*
- - [Private Voting for TCR](https://blog.enigma.co/private-voting-for-tcrs-with-enigma-b441b5d4fa7b)
- - [Consensus/PLCR implementation](https://github.com/ConsenSys/PLCRVoting)
- - [Aragon Voting App](https://wiki.aragon.org/dev/apps/voting/)
-
 ## Acknowledgement
```
Comment (lightclient) [inline EIPS/eip-1202.md:102]:
```suggestion
## Acknowledgement
```

Please also remove this section as it also invalid.

**Review by lightclient — CHANGES_REQUESTED** [review]
Apologies - I missed the "Acknowledgement" section as well.

**Comment by xinbenlv** [comment]
> Apologies - I missed the "Acknowledgement" section as well.

Per feedback, the section Acknowledgement is now removed.

File: EIPS/eip-1202.md [hunk 3]
```diff
@@ -113,32 +99,6 @@ The third consideration is non-trivialness. Some voting applications assume ***a
...
- - [Paper: A Smart Contract for Boardroom Voting with Maximum Voter Privacy](https://eprint.iacr.org/2017/110.pdf) - *Suggested by @aodhgan*
- - [Private Voting for TCR](https://blog.enigma.co/private-voting-for-tcrs-with-enigma-b441b5d4fa7b)
- - [Consensus/PLCR implementation](https://github.com/ConsenSys/PLCRVoting)
- - [Aragon Voting App](https://wiki.aragon.org/dev/apps/voting/)
-
-## Acknowledgement
-We appreciate Ansley, Andrew, Fred from Enigma, Fan and Raullen from IoTex for sharing us their use cases. we also appreciate the valuable input for designing an EIP from distinguished community members including: @frozeman, @fulldecent, @bingen, @aodhgan.
-
 ## Work Directory
 The drafting and revision of EIP-1202 is conducted at [GitHub/xinbenlv/eip-1202](https://github.com/xinbenlv/eip-1202)
```
Comment (axic) [inline EIPS/eip-1202.md:103]: @lightclient how do you feel about this section? 😬

**Comment by axic** [comment]
@xinbenlv a suggestion: in the specification consider using Solidity interfaces as opposed to contracts.

**Comment by xinbenlv** [comment]
@lightclient do you know if there are anything else I need to resolve before moving on the review?

**Review by xinbenlv — COMMENTED** [review]
Thank you @axic  for the suggestion, I updated accordingly.

**Comment by xinbenlv** [comment]
@axic Thank you for your suggestion, I updated accordingly.

File: EIPS/eip-1202.md (hunk 3 above)
Comment (MicahZoltu) [reply EIPS/eip-1202.md:103]: EIPs are specifications, not a place for storing history, lists of tools, links, etc.  A good specification is short and concise without a bunch of space wasted on stuff unnecessary for implementing the something that is spec compliant.

**Comment by axic** [comment]
> @axic Thank you for your suggestion, I updated accordingly.

I'd also suggest to that the code still compiles. Interfaces only allow `external` functions and not `public` ones. For a good example ERC see https://eips.ethereum.org/EIPS/eip-3156. It uses interfaces, NatSpec, and additional comments explaining what should happen.

**Comment by xinbenlv** [comment]
@axic will do. Let me get that validated and get back to you again. Appreciate your feedback!

**Comment by xinbenlv** [comment]
@axic by the way, I am thinking: for those `MUST`, I wonder if that would best work as a test. Is the EIP workshop considering any time soon to impose optional TESTs in a ERC so anyone who implement an ERC standard can have a canonical way to validate their standard is certain ERC-conforming, except for just matching the method signature?

File: EIPS/eip-1202.md (hunk 3 above)
Comment (xinbenlv) [reply EIPS/eip-1202.md:103]: Got it. Makes sense!

Comment (xinbenlv) [reply EIPS/eip-1202.md:103]: @MicahZoltu with the same argument, I wonder if we still want to keep "Rationale" in the standard.

**Comment by MicahZoltu** [comment]
This PR appears to wipe out the EIP entirely.  Was this a mistake?  It definitely should not be merged as is.

**Comment by xinbenlv** [comment]
Still working on revisions

**Comment by xinbenlv** [comment]
Still working on revisions

**Comment by xinbenlv** [comment]
Still working on revisions

**Comment by xinbenlv** [comment]
Still working on it

**Comment by xinbenlv** [comment]
WIP

**Comment by xinbenlv** [comment]
WIP

**Comment by xinbenlv** [comment]
wip

**Comment by xinbenlv** [comment]
@Pandapip1 can you help re-run CI?

**Review by xinbenlv — COMMENTED** [review]
All comments resolved, including latest comment from @lightclient

**Review by Pandapip1 — APPROVED** [review]
Looks good.

**Pull request merged**